generators/client/templates/vue/package.json.ejs

```diff
@@ -20,10 +20,10 @@
     "sockjs-client": "1.1.4",
     "webstomp-client": "1.2.0",
     <%_ } _%>
-    "vue": "2.5.17",
+    "vue": "2.5.21",
```

Comment:
downgrading the version to 2.5.17 works locally

Reply:
thx.